src/neptune/internal/backends/swagger_client_wrapper.py

```diff
@@ -80,6 +80,9 @@ def handle_neptune_http_errors(response, exception: Optional[HTTPError] = None):
             "WORKSPACE_IN_READ_ONLY_MODE": lambda response_body: NeptuneLimitExceedException(
```

Comment:
Let's differentiate the workspace-level and project-level exceptions

Reply:
I could also update the current `NeptuneLimitExceedException` message to be more helpful for each scenario, unless there's some technical reason for the split?

Reply:
I would say it's not related to technical, it's more about whether is it OK (from a user perspective) to use one exception for both cases.

Reply:
Yes. It's fine to have only `NeptuneLimitExceedException` as long as it covers both situations :wink:

Reply:
Alright, I think that would be fine. I'll add a proposal tomorrow.